### PR TITLE
Createdb

### PR DESCRIPTION
**Pull request by at-longhpv**
https://trello.com/b/xdp6MLNN/bookstore

Mô hình Database

https://docs.google.com/spreadsheets/d/1I5N04lbgaV6Le0Ol4jNBsDeogMkJXljqBo2j9UiGIYc/edit?pref=2&pli=1#gid=0

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -23,14 +23,10 @@ gem 'jbuilder', '~> 2.0'
...
-
-# Use Unicorn as the app server
-# gem 'unicorn'
-
-# Use Capistrano for deployment
-# gem 'capistrano-rails', group: :development
+gem 'therubyracer'
+gem 'bootstrap-sass', '~> 3.3.6'
+gem 'devise'
+gem 'simple_form', '~> 3.2'
```
Comment (at-khanhlb) [inline Gemfile]: 89565-9

**Comment by at-longhpv** [comment]
test

**Comment by at-longhpv** [comment]
testtt

**Comment by at-phunt** [comment]
abc xxxyz

**Pull request merged**